tests/job_pytest.py

```diff
@@ -1,12 +1,41 @@
 from src.job import Job
-from src.load_profile import LoadProfile
+import pandas as pd
+import pytest
 
 
 def test_to_message():
-    load_profile = LoadProfile([0.0, 3600.0], [0.0, 1.0])
+    load_profile = pd.Series(index=[0.0, 3600.0], data=[0.0, 1.0])
     job = Job(0, 1, load_profile)
 
     expected = dict(est=0, lst=1, load_profile=load_profile)
     actual = job.to_message()
 
     assert expected == actual
+
+def test_power():
+    job1 = Job(1, 2, pd.Series(index=[0.0, 1.0, 3.0, 4.0], data=[0.0, 10.0, 30.0, 40.0]))
+    job2 = Job(0, 1, pd.Series(index=[0.0, 3600.0], data=[0.0, 1.0]))
+
+    expected1 = 0.0
+    expected2 = 0.0
+    expected3 = pytest.approx(15.0, 0.1)

```

Comment:
these values are only approximately equal. `pytest.approx(expected, rel)` is a handy function for when you only approximately know what number to test against. `rel` tells how big the difference can be for the test to pass.